v0.0.1/model-schema.json

```
@@ -0,0 +1,192 @@
+            "format": "email"
+          },
+          "twitter": {
+            "type": "string"
+          },
+          "additionalProperties": false
+        }
+      }
+    },
+    "website_url": {
+      "description": "Public facing website for the model",
+      "type": "string",
+      "format": "uri"
+    },
+    "repo_url": {
+      "description": "Repository containing code for the model",
+      "type": "string",
+      "format": "uri"
+    },
+    "license": {
+      "description": "License for use of model output data",
+      "type": "string",
+      "enum": [
+        "apache-2.0",
+        "cc-by-4.0",
+        "cc-by-nc-4.0",
+        "cc-by-nc-nd-4.0",
+        "cc-by-sa-4.0",
+        "gpl-3.0",
+        "lgpl-3.0",
+        "mit"
+      ]
+    },
+    "include_viz": {
+      "description": "Indicator for whether the model should be included in the Hub’s visualization",
+      "type": "boolean"
+    },
+    "include_ensemble": {
+      "description": "Indicator for whether the model should be included in the Hub’s ensemble",
+      "type": "boolean"
+    },
+    "include_eval": {
+      "description": "Indicator for whether the model should be scored for inclusion in the Hub’s evaluations",
+      "type": "boolean"
+    },
+    "citation": {
+      "type": "string"
+    },
+    "team_funding": {
+      "type": "string"
+    },
+    "model_details": {
+      "type": "object",
+      "properties": {
+        "data_inputs": {
+          "type": "string"
+        },
+        "methods": {
+          "type": "string",
+          "maxLength": 200
+        },
+        "methods_long": {
+          "type": "string"
+        }
+      },
+      "additionalProperties": false,
+      "required": ["data_inputs", "methods"]
+    },
+    "ensemble_of_hub_models": {
+      "description": "Indicator for whether this model is an ensemble of other Hub models",
+      "type": "boolean"
+    },
+    "modeling_NPI": {
+      "type": "string"
+    },
+    "compliance_NPI": {
+      "type": "string"
+    },
+    "contact_tracing": {
+      "type": "string"
+    },
+    "testing": {
+      "type": "string"
+    },
+    "vaccine_efficacy_transmission": {
+      "type": "string"
+    },
+    "vaccine_efficacy_delay": {
+      "type": "string"
+    },
+    "vaccine_hesitancy": {
+      "type": "string"
+    },
+    "vaccine_immunity_duration": {
+      "type": "string"
+    },
+    "natural_immunity_duration": {
+      "type": "string"
+    },
+    "case_fatality_rate": {
+      "type": "string"
+    },
+    "infection_fatality_rate": {
+      "type": "string"
+    },
+    "asymptomatics": {
+      "type": "string"
+    },
+    "age_groups": {
+      "type": "string"
+    },
+    "importations": {
+      "type": "string"
+    },
+    "confidence_interval_method": {
+      "type": "string"
+    },
+    "calibration": {
+      "type": "string"
+    },
+    "spatial_structure": {
+      "type": "string[bc]"
+    }
+  },
+  "additionalProperties": false,
```

I suggest removing all properties starting with "modeling_NPI" and below, and changing additionalProperties to true.
```suggestion
  "additionalProperties": true,
```

---

This feels related to me to the question I raised in a comment below about what the purpose of this file is. I agree that we don't want to put ourselves in a position of listing all possible model metadata fields, so setting `"additionalProperties":` true makes sense. But we have discussed that we might like to herd hubs toward a common set of terminology and fields where possible. For example, this is what we did with task id variables: we agreed on a list of common task id variables that many hubs might like to use, and listed them in the tasks-schema.json file along with the setting that additional properties could be listed. Representatives from the scenario modeling hubs have expressed an interest in similarly standardizing the model metadata.